ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/platform.j2

```diff
@@ -27,6 +27,9 @@ platform sand forwarding mode {{ platform.sand.forwarding_mode }}
 {%         if platform.sand.multicast_replication.default is arista.avd.defined %}
 platform sand multicast replication default {{ platform.sand.multicast_replication.default }}
 {%         endif %}
+{%         if platform.sand.mdb.profile is arista.avd.defined %}
+platform sand mdb profile {{ platform.sand.mdb.profile }}
```

Comment:
Have you checked the EOS config order? Just resolve if yes :)

Reply:
I have check this on a physical box and it appears correct. Do we want to also match the `!` used for spacing?

![image](https://github.com/aristanetworks/ansible-avd/assets/109683698/af2f5cb1-ef59-4f78-85d6-8b2c37cdf6ac)